.travis.yml

```diff
@@ -28,6 +28,7 @@ script:
   elif [ "$TEST_TYPE" = js ]
   then
 
+    cat <(echo eslint; npm run lint --silent -- --format=json; echo flow; flow --json) | GITHUB_TOKEN="af6ef0d15709bc91d""06a6217a5a826a226fb57b7" node bots/code-analysis-bot.js
```

Comment:
Unfortunately, there's no way to have a private token on an untrusted pull request. Just put broke it into two parts so that it makes mass collection harder I guess.